pkg/providerConfigs/constants.go

```diff
@@ -19,8 +19,11 @@ const (
 	LocalhostOS            = runtime.GOOS
 	LocalhostArch          = runtime.GOARCH
 	TerraformClientVersion = "1.3.8"
-	ArgocdHelmChartVersion = "4.10.5"
 
 	ArgocdPortForwardURL = runtimepkg.ArgocdPortForwardURL
 	VaultPortForwardURL  = runtimepkg.VaultPortForwardURL
+
+	TokenRegexPattern = "<([A-Z_0-9-]+)>"
+	leftDelimiter     = "${k1"
+	rightDelimiter    = "}"
```

Comment:
After discussing with the rest of the team, we want to go with these for delimiters:

```suggestion
	leftDelimiter     = "<%"
	rightDelimiter    = "%>"
```